service-tests/nsp.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const Joi = require('joi')
+const ServiceTester = require('./runner/service-tester')
+
+const t = new ServiceTester({id: 'nsp', title: 'Node Security Platform'})
+
+t.create('get a package without vulnerabilities')
+  .get('/npm/bronze.json')
+  .expectJSONTypes(Joi.object().keys({name: 'nsp', value: 'no known vulnerabilities'}))
+
+t.create('get a package with vulnerabilities')
+  .get('/npm/nodeaaaaa.json')
+  .expectJSONTypes(Joi.object().keys({name: 'nsp', value: Joi.string().regex(/^[0-9]+ vulnerabilities$/)}))
+
+t.create('get a package that does not exist')
+  .get('/npm/some-unknown-package.json')
+  .expectJSONTypes(Joi.object().keys({name: 'nsp', value: 'n/a'}))
```

Comment:
Is it possible to distinguish between **package not found** and a server error? In the first case, it would be good to provide a more helpful error message. In the second case, would you please use **invalid**, which is consistent with the other badges?

Reply:
That's fair. Any preferred `colorscheme` for each? How would you recommend catching server errors?

Reply:
I'd suggest figuring out what a response looks like for a non-existent package, and testing for that specifically. Once you've identified and excluded that case, if `available === false` I'd say that's a server error.

Most of the other badges use the default color for errors.